Examples/StereoKitTest/Tests/TestNodes.cs

```diff
@@ -6,6 +6,10 @@ class TestNodes : ITest
 	Pose      _modelPose = Pose.Identity;
 	Model     _model;
 	ModelNode _surfaceNode;
+	ModelNode modelRay2Node;
+	ModelNode modelRay3Node;
+	ModelNode ray2ModelNode;
+	ModelNode ray3ModelNode;
```

Comment:
C#'s official naming convention has private fields such as these starting with and underscore, so `_modelRay2Node`, etc.